job-node.yaml

```diff
@@ -10,6 +10,9 @@ spec:
       - name: kube-bench
         image: aquasec/kube-bench:latest
         command: ["kube-bench","node"]
+        env:
+          - name: PATH
+            value: /bin:/usr/bin:/usr/local/bin
```

Comment:
On further reflection, I'm wondering if the modified PATH shouldn't include the locations that are currently defined in the image: 
```
$ docker run --rm -it --entrypoint=/bin/sh aquasec/kube-bench
/opt/kube-bench # echo $PATH
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Wdyt?

Reply:
@lizrice I've been investigating this issue as it's been causing us trouble as well. We're on EKS using the AWS-provided image as well.

I think the issue comes from the fact that the container mounts `/usr/bin` from the host into `/usr/bin` within kube-bench, and a `ps` happens to be available in there. Well, _available_... Kind of:
```
/opt/kube-bench # ls -lah /usr/bin/ps
-rwxr-xr-x    1 root     root       97.8K Aug  1  2018 /usr/bin/ps
/opt/kube-bench # /usr/bin/ps
/bin/sh: /usr/bin/ps: not found
```
I think that by virtue of existing (at least to `ls`) and being earlier in the path, this `ps` gets picked up before the "real" `ps` from the container at `/bin/ps`:
```
/opt/kube-bench # ps
  PID TTY          TIME CMD
20719 pts/0    00:00:00 sh
23531 pts/0    00:00:00 ps
/opt/kube-bench # /bin/ps
  PID TTY          TIME CMD
20719 pts/0    00:00:00 sh
23535 pts/0    00:00:00 ps
```
But because it's a program from the host, some protection mechanism kicks in and prevents kube-bench from executing it, hence the error message from #220:
```
I0219 11:26:28.944632   20794 util.go:57] [ps -C hyperkube -o cmd --no-headers]: fork/exec /usr/bin/ps: no such file or directory
I0219 11:26:28.944662   20794 util.go:284] executable 'hyperkube kubelet' not running
```

I'm not sure what the best solution is here, to be honest. Potentially just rearrange the path to keep all the existing entries, but have `/bin` be a higher priority than `/usr/bin`? This works for me for example:
```yaml
env:
- name: PATH
  value: /usr/local/sbin:/usr/local/bin:/usr/sbin:/sbin:/bin:/usr/bin
```

Reply:
I like the look of @Pluies solution here 
```
env:
- name: PATH
  value: /usr/local/sbin:/usr/local/bin:/usr/sbin:/sbin:/bin:/usr/bin